roles/openshift_efk/README.md

```diff
@@ -0,0 +1,9 @@
+###Required vars:
+
+- kibana_hostname: kibana.example.com
+- es_cluster_size: 1
+- master_url: https://localhost:8443
```

Comment:
Recommended best practices for role variable naming is to use <role_name>_<var_name> to avoid variable name collisions.

I would suggest the following names instead (assuming a rename of the role to openshift_logging):
```
openshift_logging_hostname
openshift_logging_elasticsearch_cluster_size
openshift_logging_master_public_url
openshift_logging_secret_vars
openshift_logging_fluentd_replicas